Blocks/URL/url.h

```diff
@@ -230,11 +231,8 @@ struct URLParametersExtractor {
   static bool IsURIComponentSpecialCharacter(char c) {
     // https://github.com/lyokato/cpp-urilite/blob/723083f98ddf42f2b610b62444c598236b5ce3e8/include/urilite.h#L52-L65
     // RFC2396 unreserved?
-    return ((c >= 'a' && c <= 'z')
-            || (c >= 'A' && c <= 'Z')
-            || (c >= '0' && c <= '9')
-            || c == '-'  || c == '_'  || c == '.' || c == '~'
-            || c == '!'  || c == '\'' || c == '(' || c == ')');
+    return ((c >= 'a' && c <= 'z') || (c >= 'A' && c <= 'Z') || (c >= '0' && c <= '9') || c == '-' || c == '_' ||
+            c == '.' || c == '~' || c == '!' || c == '\'' || c == '(' || c == ')');
```

Comment:
This is one more example where the shape of the code is broken by an auto-indent.